E-Commerce/Controllers/AdminController.cs

```diff
@@ -12,10 +14,12 @@ public class AdminController : Controller
     {
         private readonly IDashboardRepository dashboard;
 
-       public AdminController(IDashboardRepository dashboard)
+
+        public AdminController(IDashboardRepository dashboard )
         {
             this.dashboard = dashboard;
         }
+
```

Comment:
Consider locking down the admin dashboard further by requiring authorization for the`Index()` action